### ISSUE TITLE
Moving tasks between projects

### ISSUE THREAD

**Issue by ilya-sha** [cvat-ai/cvat#8315]
### Actions before raising this issue

- [X] I searched the existing issues and did not find anything similar.
- [X] I read/searched [the docs](https://docs.cvat.ai/docs/)

### Is your feature request related to a problem? Please describe.

I would like to move some tasks to identical project I have created (same label list and attributes).

> I use the task options in Django to move the task.
> ![image](https://github.com/user-attachments/assets/dcbd44c1-1565-42c9-92e5-53b0b05e902c)

When I transfer a task from one project to the other, it moves. I can see it in the new project task list, but I'm not able to open the jobs.
When I transfer it back to the original project, I'm able to open it as normal.

### Describe the solution you'd like

I would like to have an easy, user friendly, option to move multiple tasks in a batch from one project to the other.
* Matching label list

**Comment by gboeer** [comment]
Indeed, it doesn't seem to be possible to move a task to another project or to be more specific it's possible to move the task, but upon opening it afterwards an error is displayed.
